docs/htdocs/info/docs/tools/vep/script/vep_options.html

```diff
@@ -1873,39 +1879,67 @@ <h2 id="filt">Filtering and QC options</h2>
               </tr>
 
               <tr class="bg1">
-                <td>AA</td><td>NHLBI-ESP African American</td>
+                <td>gnomADe</td><td>gnomAD exomes combined population</td>
               </tr>
               <tr>
-                <td>EA</td><td>NHLBI-ESP European American</td>
+                <td>gnomADe_AFR</td><td>gnomAD exomes African/African American population</td>
+              </tr>
+              <tr class="bg1">
```

Comment:
In this table, there are some rows that are coloured in light grey. What is special about those rows? Couldn't we highlight the values regarding global / combined population instead, e.g. gnomADg, gnomADe and 1KG_ALL?